Sources/Swift/Exports.swift

```diff
@@ -0,0 +1,3 @@
+#if SENTRY_SWIFT_PACKAGE
```

Comment:
`m`: A header comment with documentation of the file purpose would be convenient.